osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -360,6 +360,14 @@ private void setState(LeaderboardState state)
             }
         }
 
+        /// <summary>
+        /// Whether the leaderboard is displaying scores (at least one).
+        /// </summary>
+        public bool HasScores()
+        {
+            return state == LeaderboardState.Success;
+        }
```

Comment:
I don't think there's much point to this method. I'd just expose `state` as publicly readable (but _not_ settable) at this point.